Releases.md

```diff
@@ -6,6 +6,29 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at
 https://github.com/denoland/deno_install
 
+### 1.0.1 / 2020.05.19
+
+- feat(std/node): fs.writeFileSync polyfill (#5414)
+- feat: add std/hash/fnv (#5403)
+- fix(std/hash): SHA1 hash of Uint8Array (#5086)
+- fix(std/node): fs.readFile should take string as option (#5316)
+- fix(std/testing): Provide message and diff for assertStrictEq (#5417)
```

Comment:
If `std` isn't going to be bumped at the same time (?), these should be listed later under `### std/0.52.0`.

Reply:
we might as well bump std too